### DISCUSSION TITLE
Android port

### DISCUSSION THREAD

**Discussion by enese94** [Alex313031/Mercury#145] [General]
Will you ever do a port or Mercury to Android in the same way you did to Thorium? Mercury is so much faster than vanilla Firefox. If you could do the same thing to android, it would be awesome.

**Comment by SyCoREAPER** [comment]
@Alex313031

Not a demand as you are doing this in your own time and resources but an Android variant would be welcome.

While I honestly would prefer to use your Thorium build, the Android version as you (obviously) know has no extension support while Mercury could at least be a viable alternative.

**Comment by MarcoRavich** [comment]
Well, evaluate a collaboration with @Divested-Mobile's [Mull](https://github.com/Divested-Mobile/mull-fenix#readme) then.

**Reply by SyCoREAPER** [reply]
Mull, Fennec and Ice Raven are all pretty good. But Alex's builds, at least on a perceived level, runs laps around them in terms of speed feel. Unfortunately the updates just aren't there to make it a DD, same with Thorium. Leaps and bounds faster but the updates just don't happen. 

I don't blame the dev it is provided free after all and in their own free time.

**Comment by SyCoREAPER** [comment]
For @Alex313031 
Bumping this to your notifications to show continued interest in a port. 

I'm dying for an Android port. I know you are already swamped with Thorium PC, Android, OS, and Mercury Desktop.

I'd love to see this beast on mobile. I love your Thorium and would use that but Google being Google means no extensions ☹️ as where this would.
